docs/timingmodels.rst

```diff
@@ -0,0 +1,19 @@
+.. _`Timing Models`:
+
+Timing Models
+=============
+
+This section is to explain the bulit-in timing models.
+
+PINT built-in timing models are located at the directory PINT/src/pint/models_.
+Timing model is structured as model components which implement difference pieces
+of timing model, and the parent ``TimingModel`` object for overall model components
```

Comment:
You can make Sphinx recognize this and form a link to the reference page for TimingModel; I believe the syntax is 
```
:class:`pint.models.timing_model.TimingModel`
```
There is documentation there it would be good to point to.

Reply:
I think it gets linked properly now